core/method.rbs

```diff
@@ -161,7 +168,7 @@ class Method < Object
   #     m.call # => "bar"
   #     n = m.clone.call # => "bar"
   #
-  def clone: () -> Method
+  def clone: () -> instance
```

Comment:
`clone`, like `dup`, technically return `instance`s, but there's no actual way to test this (you can't create subclasses of `Method` to call `#clone` on)